README.md

```diff
@@ -33,6 +33,9 @@
    - [Open Source Initiative](https://opensource.org)
    - [The Linux Foundation](https://www.linuxfoundation.org)
    - [GNOME Community](https://wiki.gnome.org/Community)
+   - [EddieHub](https://github.com/EddieHubCommunity)
+   - [Oss Cameroun](https://github.com/osscameroon)
+   - [Devicons](https://github.com/devicons)
```

Comment:
I don't think this is an OSS organization. Kindly remove this line from the list; thank you.

Reply:
Ok

Reply:
i did it